src/groups/Command.js

```diff
@@ -72,6 +72,8 @@ module.exports.Command = class Command {
 		if (this.perms.aldebaran !== undefined && check)
 			check = this.perms.aldebaran
 				.every(perm => message.author.hasPermission(perm));
+		if (message.author.id === process.env.BOT_ADMIN)
```

Comment:
I think the permissions check should not be done here, it should be in the User structure.